lib/Controllers/list.js

```diff
@@ -95,8 +95,12 @@ List.prototype.fetch = function(req, res, context) {
     return result;
   }, {});
 
-  if (Object.keys(extraSearchCriteria).length)
-    criteria = _.assign(extraSearchCriteria, criteria);
+  if (Object.keys(extraSearchCriteria).length) {
+    if (Object.keys(criteria).length)
+      criteria = Sequelize.and(criteria, Sequelize.or.apply(null, [extraSearchCriteria]));
+    else
+      criteria = Sequelize.or.apply(null, [extraSearchCriteria]);
```

Comment:
yeah, this is actually just a brain fart on my part.. for some reason I seem to always use _.assign backwards. The actual fix for this is to reverse the order in assign to:

```
_.assign(criteria, extraSearchCriteria);
```

I can merge the fix if you'd like, or you can update the PR?